### ISSUE TITLE
controls not defined

### ISSUE THREAD

**Issue by ottenhoff** [SedueRey/ckeditor-html5-video#3]
This code seems wrong?

`          if (config.controls) {
            this.setData('controls', controls);
          }
`

**Comment by SedueRey** [comment]
yes... and those until line 102, I'm going to do the PR right now

**Comment by SedueRey** [comment]
I've created a new tag, @ottenhoff. I don't know when the webjar will be released.

**Comment by ottenhoff** [comment]
Thank you!!
